packages/devtools_app/lib/src/service/service_extension_manager.dart

```diff
@@ -24,7 +24,7 @@ import 'vm_service_wrapper.dart';
 class ServiceExtensionManager extends Disposer {
   ServiceExtensionManager(this._isolateManager);
 
-  VmServiceWrapper _service;
+  VmServiceWrapper? _service;
```

Comment:
this can be late and non-nullable

Reply:
I am not sure. _callServiceExtension checks it for null in the first line.

Reply:
I see. Leave as is then to be safe.